framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -181,10 +199,13 @@
           List<File> testQueryFiles = searchFiles(testDefFile.getParentFile(),
                   queryFileExtension);
           for (File testQueryFile : testQueryFiles) {
-            String expectedFileName = getExpectedFile(testQueryFile.getAbsolutePath(),
-            		queryFileExtension, expectedFileExtension);
-            drillTestCases.add(new DrillTestCase(modeler, testQueryFile.getAbsolutePath(), expectedFileName));
-          }
+
+	    String expectedFileName = getExpectedFile(testQueryFile.getAbsolutePath(),modeler.matrices.get(0).inputFile,
+                      expectedFileExtension);
+	    if(expectedFileName!=null && expectedFileName!=""){
+            	drillTestCases.add(new DrillTestCase(modeler, testQueryFile.getAbsolutePath(), expectedFileName));
```

Comment:
I dont know this behavior but whats the point to run the test for this ,We can check how it behaves